neo/io/basefromrawio.py

```diff
@@ -107,6 +107,9 @@ def read_block(self, block_index=0, lazy=False, signal_group_mode=None,
 
         if signal_group_mode is None:
             signal_group_mode = self._prefered_signal_group_mode
+            if self._prefered_signal_group_mode == 'split-all':
+                self.logger.warning("the default signal_group_mode will'\
+                        'be 'split-all' in next release")
```

Comment:
In #710, you said you want to change the default to ``group-by-same-units``, right? Isn't this warning wrong then?

Reply:
Maybe you should also say "will change from 'split-all' to 'group-by-same-units' in the next release". Otherwise it isn't clear to non-expert users how they will be affected.